.travis.yml

```diff
@@ -0,0 +1,6 @@
+language: java
+script: 
+- ./gradlew build
+sudo: required
+jdk:
+  - oraclejdk8
```

Comment:
Tried restarting the build to test caching. It seems like it worked, since it didn't have to download dependencies again.